pkg/deploy/generator/scripts/devProxyVMSS.sh

```diff
@@ -1,3 +1,6 @@
+echo "running RHUI fix"
+yum update -y --disablerepo='*' --enablerepo='rhui-microsoft-azure*'
+
 yum -y -x WALinuxAgent -x WALinuxAgent-udev update
 yum -y install docker
```

Comment:
why do we install plain docker here instead of podman docker like the others?

Reply:
Historically we found out, that podman is not 1to1 replacement and our deployment did not worked.

Reply:
I only moved the RP and gateway over to using podman when doing the [RHEL 8 migration](https://github.com/Azure/ARO-RP/pull/2518), but I don't think we actually use this devProxy anymore either. I simply changed it to be consistent across all the scripts.